client/pom.xml

```diff
@@ -20,7 +20,7 @@
   <parent>
     <artifactId>navigator-sdk</artifactId>
     <groupId>com.cloudera.navigator</groupId>
-    <version>1.2-SNAPSHOT</version>
+    <version>2.0</version>
```

Comment:
I have run all the examples except the HIVE example. The adding tags to Hive entities since we are planning to support other entity types in the next version